src/users/src/users-service/routes.go

```diff
@@ -47,6 +49,30 @@ var routes = Routes{
 		"/users/identityid/{identityID}",
 		UserShowByIdentityId,
 	},
+	Route{
+		"GetFilteredUser",
+		"GET",
+		"/users/filter/",
+		GetFilteredUser,
+	},
+	Route{
+		"GetRandomUser",
+		"GET",
+		"/users/getRandomUser/",
```

Comment:
I'd prefer something like `GET /users/random?count=X` for consistency where count defaults to `1`.

Reply:
Makes sense. Updated